openmetadata-ui/src/main/resources/ui/src/components/Glossary/GlossaryColumnsSelectionDropdown/DraggableMenuItem.component.tsx

```diff
@@ -0,0 +1,78 @@
+/*
+ *  Copyright 2024 Collate.
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+import { Checkbox } from 'antd';
+import React from 'react';
+import { useDrag, useDrop } from 'react-dnd';
+import { ReactComponent as ColumnDragIcon } from '../../../assets/svg/menu-duo.svg';
+
+interface DraggableMenuItemProps {
+  option: { value: string; label: string };
+  index: number;
+  moveDropdownMenuItem: (fromIndex: number, toIndex: number) => void;
+  tempCheckedList: string[];
```

Comment:
```suggestion
  selectedOptions: string[];
```

Reply:
Done, changed the tempCheckedList variable name to selectedOptions.